src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -107,26 +107,29 @@ export class HuiStackCardEditor extends LitElement
                         : "ui.panel.lovelace.editor.edit_card.show_visual_editor"
                     )}
                   </mwc-button>
-                  <ha-icon-button
-                    id="move-before"
-                    title="Move card before"
-                    icon="hass:arrow-left"
+
+                  <mwc-icon-button
                     .disabled=${selected === 0}
-                    @click=${this._handleMove}
-                  ></ha-icon-button>
+                    label="Move card before"
+                    @click=${() => this._handleMove(-1)}
+                  >
+                    <ha-svg-icon .path=${mdiArrowLeft}></ha-svg-icon>
+                  </mwc-icon-button>
 
-                  <ha-icon-button
-                    id="move-after"
-                    title="Move card after"
-                    icon="hass:arrow-right"
+                  <mwc-icon-button
+                    label="Move card after"
                     .disabled=${selected === numcards - 1}
-                    @click=${this._handleMove}
-                  ></ha-icon-button>
+                    @click=${() => this._handleMove(+1)}
```

Comment:
You should just make this a property on the icon-button

Reply:
```suggestion
					.move=${1}
                    @click=${this._handleMove}
```

And then handle this in the function

Reply:
Done